sensu/transport/rabbitmq/rabbitmq.go

```diff
@@ -2,30 +2,86 @@ package rabbitmq
 
 import (
 	"errors"
-	"github.com/upfluence/sensu-go/Godeps/_workspace/src/github.com/upfluence/goutils/log"
+	"fmt"
+	"math/rand"
+	"time"
 
-	"github.com/upfluence/sensu-go/Godeps/_workspace/src/github.com/streadway/amqp"
+	"github.com/upfluence/sensu-client-go/Godeps/_workspace/src/github.com/streadway/amqp"
```

Comment:
Ouch... Not sure how I messed that one up. I'll blame it on goimports for now and pay more attention next time.